### PR TITLE
Create a form to add judges

### PR DESCRIPTION
**Pull request by robmcl4**

### CODE REVIEW

File: frontend/src/Admin/components/CreateJudgeForm.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    create(values)
+  }
+
+  render () {
+    const {
+      handleSubmit
+    } = this.props
+
+    return (
+      <Form onSubmit={handleSubmit(this.submit)} style={{marginBottom: '25px', marginTop: '25px'}}>
```
Comment (robmcl4) [inline frontend/src/Admin/components/CreateJudgeForm.js]: This is probably bad practice

File: frontend/src/Admin/components/CreateJudgeForm.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react'
+import { Form, FormGroup, Label, Button } from 'reactstrap'
+import { Flex, Box } from 'rebass'
+import { Field, reduxForm } from 'redux-form'
+
+class CreateJudgeForm extends Component {
+  submit = (values) => {
+    const { create } = this.props
+
+    create(values)
```
Comment (robmcl4) [inline frontend/src/Admin/components/CreateJudgeForm.js]: I want to clear the redux form when this is done

File: frontend/src/Admin/containers/CreateJudgeForm.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    myAddJudge: (judge) => dispatch(addJudge(judge))
+  }
+}
+
+const withMutations = graphql(CreateJudgeMutation, {
+  props: ({mutate, ownProps: {myAddJudge}}) => ({
+    create: (judge) => mutate({
+      variables: { input: judge }
+    }).then(({data: {createJudge}}) => myAddJudge(createJudge))
+    // TODO catch errors
```
Comment (robmcl4) [inline frontend/src/Admin/containers/CreateJudgeForm.js]: plz

**Comment by billdybas** [comment]
I made some changes - see my commits.

Otherwise, LGTM.

**Pull request merged**